filter_aws: add more IMDS categories (#1530)
fluent/fluent-bit-docs: pipeline/filters/aws-metadata.md

| Key | Description | Default |
| :--- | :--- | :--- |
| imds\_version | Specify which version of the instance metadata service to use. Valid values are 'v1' or 'v2'. | v2 |

> | partition | The [partition](https://docs.aws.amazon.com/whitepapers/latest/aws-fault-isolation-boundaries/partitions.html); for example, "aws". | false |

Reviewer: ```suggestion
| partition | The [partition](https://docs.aws.amazon.com/whitepapers/latest/aws-fault-isolation-boundaries/partitions.html), such as `"aws"`. | false |
```

> | domain | The domain for AWS resources in the region; for example, "amazonaws.com". | false |

Reviewer: ```suggestion
| domain | The domain for AWS resources in the region, such as `"amazonaws.com"`. | false |
```

> | region | The [region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions); for example, "us-east-1". | false |

Reviewer: ```suggestion
| region | The [region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-regions), such as `"us-east-1"`. | false |
```

> | az | The [availability zone](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones); for example, "us-east-1a". | true |

Reviewer: ```suggestion
| az | The [availability zone](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones), such as `"us-east-1a"`. | true |
```

> | az\_id | The [availability zone ID](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#az-ids); for example, "use1-az1". | false |

Reviewer: ```suggestion
| az\_id | The [availability zone ID](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#az-ids), such as `"use1-az1"`. | false |
```

```text
[INPUT]
  Name              dummy
  Tag               dummy

[FILTER]
  Name              aws
  Match             *
  Imds_Version      v1
  Partition         true
  Az                true
  Domain            true
  Region            true
  Az_Id             true
  Ec2_Instance_Id   true
  Ec2_Instance_Type true
  Placement_Group   true
  Partition_Number  true
  Host_Id           true
  Ec2_Instance_Id   true

>   Ec2_Instance_Type true

Reviewer 2: fixing.

  Ipv6              true
  Private_Ip        true

>   Public_Ip         true  

Reviewer 2: Fixing.
